server/services/approverCaseloadService.ts

```diff
@@ -41,64 +41,32 @@ export default class ApproverCaseloadService {
       filterCentralCaseload(prisonCaseload),
       user
     )
-    const caseLoad: ManagedCaseForApproval[] = await this.mapLicencesToOffenders(licences)
-    return this.applySearch(searchString, await this.mapResponsibleComsToCases(caseLoad))
+    const caseLoad: ApprovalCase[] = await this.mapLicencesToOffenders(licences)
+    return this.applySearch(searchString, caseLoad)
   }
 
   async getRecentlyApproved(user: User, prisonCaseload: string[], searchString: string): Promise<ApprovalCase[]> {
     const licences: LicenceSummaryApproverView[] = await this.licenceApiClient.getLicencesRecentlyApproved(
       filterCentralCaseload(prisonCaseload),
       user
     )
-    const caseLoad: ManagedCaseForApproval[] = await this.mapLicencesToOffenders(licences)
-    return this.applySearch(searchString, await this.mapResponsibleComsToCases(caseLoad))
+    const caseLoad: ApprovalCase[] = await this.mapLicencesToOffenders(licences)
+    return this.applySearch(searchString, caseLoad)
   }
 
   private mapLicencesToOffenders = async (
     licences: LicenceSummaryApproverView[],
     user?: User
-  ): Promise<ManagedCaseForApproval[]> => {
+  ): Promise<ApprovalCase[]> => {
     const nomisIds = licences.map(l => l.nomisId)
     const deliusRecords = await this.communityService.getOffendersByNomsNumbers(nomisIds)
-    const offenders = await this.pairDeliusRecordsWithNomis(deliusRecords, user)
-    return offenders.map(offender => {
-      return {
-        ...offender,
-        licences: licences
-          .filter(l => l.nomisId === offender.nomisRecord.prisonerNumber)
-          .map(l => {
-            const releaseDate = l.actualReleaseDate || l.conditionalReleaseDate
-            return {
-              id: l.licenceId,
-              type: <LicenceType>l.licenceType,
-              status: <LicenceStatus>l.licenceStatus,
-              comUsername: l.comUsername,
-              dateCreated: l.dateCreated,
-              approvedBy: l.approvedByName,
-              approvedDate: l.approvedDate,
-              versionOf: l.versionOf,
-              kind: <LicenceKind>l.kind,
-              submittedByFullName: l.submittedByFullName,
-              isDueForEarlyRelease: l.isDueForEarlyRelease,
-              isDueToBeReleasedInTheNextTwoWorkingDays: l.isDueToBeReleasedInTheNextTwoWorkingDays,
-              releaseDate: releaseDate ? parseCvlDate(releaseDate) : null,
-            }
-          }),
-      }
-    })
-  }
-
-  private pairDeliusRecordsWithNomis = async (
-    managedOffenders: DeliusRecord[],
-    user: User
-  ): Promise<ManagedCaseForApproval[]> => {
-    const caseloadNomisIds = managedOffenders
-      .filter(offender => offender.otherIds?.nomsNumber)
+    const caseloadNomisIds = deliusRecords
       .map(offender => offender.otherIds?.nomsNumber)
+      .filter(nomsNumber => Boolean(nomsNumber))
```

Comment:
just reordered these to reduce occurrences where we deeply access internal of a bit of data